Installed/Driver/Sound/SoundBlaster/Makefile

```diff
@@ -3,7 +3,7 @@
 #
 # If you edit it, you will lose your changes, should it be regenerated.
 #
-GEODE           = sblaster

```

Comment:
Changing the geode name is likely not a good idea here? Could we revert this change?